src/webpack/create-controllers-module.js

```diff
@@ -46,14 +49,23 @@ module.exports = function createControllersModule(config) {
             const controllerMain = packageName + '/' + controllerPackageConfig.main;
             const webpackMode = controllerUserConfig.webpackMode;
 
-            controllerContents +=
-                "\n  '" +
-                controllerReference +
-                '\': import(/* webpackMode: "' +
-                webpackMode +
-                '" */ \'' +
-                controllerMain +
-                "'),";
+            let moduleValueContents = `import(/* webpackMode: "${webpackMode}" */ '${controllerMain}')`;
+            if (webpackMode === 'lazy-controller') {
+                hasLazyControllers = true;
+                moduleValueContents = `
+new Promise((resolve, reject) => resolve({ default:
+      class extends Controller {
+          initialize() {
+              import('${controllerMain}').then(controller => {
+                  this.application.register('${controllerNormalizedName}', controller.default);
+              });
+          }
+      }
+  }))
```

Comment:
Because this runs after babel, we need to rewrite this into "old" JavaScript to support all browsers. I have the code locally, just need to put it in here :)

Reply:
or maybe we can make this run before babel

Reply:
Yea... but I could not find a way to do that :/. We effectively would need to change the "filename" of the output source to something that ends in `.js` and get Webpack to, sort of, "restart" the process so that the normal .js loader can match it. That may be possible, but I couldn't find away. And... really, we ship "dist" code normally that has been pre-compiled with Babel. This just a special case of doing the same (the strangeness being that I'll rewrite this to ES5 manually).